plex_trakt_sync/commands/unmatched.py

```diff
@@ -5,17 +5,24 @@
 from plex_trakt_sync.factory import factory
 from plex_trakt_sync.walker import Walker
 
-
```

Comment:
restore two newlines here. some PEP style.

Reply:
yeah sorry i missed that! done